Library/Homebrew/caveats.rb

```diff
@@ -108,13 +108,23 @@ def function_completion_caveats(shell)
     end
   end
 
+  def python
+    @python ||= if OS.mac?
+      "python"
+    else
+      which("#{HOMEBREW_PREFIX}/opt/python/bin/python2") ||
+        which("python") || which("python2") || which("python2.7")
```

Comment:
Try `which("python").to_s` and the like. `which("stuff")` returns a Pathname object

Reply:
should this be in `extend/os/linux/caveats.rb`?